apps/aecore/lib/aecore/pow/cuckoo.ex

```diff
@@ -135,7 +135,7 @@ defmodule Aecore.Pow.Cuckoo do
         handle_raw_data(process, buff)
       any ->
         Logger.error("[Cuckoo] Unexpeted error : #{inspect(any)}")
-        exit(:kill)
+        wait_for_result(process, buff)
```

Comment:
why is this changed?

Reply:
Ivan suggested this as a workaround, we seem to get an unexpected message here.

Reply:
ok great